.gitignore

```diff
@@ -9,4 +9,4 @@ sunspot_solr/solr/solr-webapp
 .yardoc
 docs
 *.swp
-.idea
+sunspot/spec/solr/
```

Comment:
I bet you forgot to upload some specs because of this line 😉  Is this really needed?